tasks/go_test.py

```diff
@@ -70,7 +70,7 @@ def ensure_bytes(s):
     return s
 
 
-def build_stdlib(
+def build_standard_lib(
```

Comment:
Why changing the name? I feel like using `stdlib` was more accurate and coherent with your new flag. Otherwise LGTM

Reply:
Because otherwise the flag and the function had the same name which was buggy ! So I had 2 names `build_standard_lib` and `build_stdlib` and I chose the shortest to be the flag (since we'll type it more often in the cli).